src/shims/windows/foreign_items.rs

```diff
@@ -141,61 +141,100 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         // https://github.com/rust-lang/rust/blob/fb00adbdb69266f10df95a4527b767b0ad35ea48/compiler/rustc_target/src/spec/mod.rs#L2766-L2768,
         // x86-32 Windows uses a different calling convention than other Windows targets
         // for the "system" ABI.
-        let sys_conv = if this.tcx.sess.target.arch == Arch::X86 {
-            CanonAbi::X86(X86Call::Stdcall)
-        } else {
-            CanonAbi::C
-        };
+        // TODO: Use this in shim-sig instead of 'system'? Or is that good enough for ExternAbi?
```

Comment:
Drawing attention to this - no tests are failing locally, but I need to double check whether this works, or if I need to make the `shim_sig` macro support dynamic ABIs.

Reply:
"system" was added exactly to support the Windows API, so `extern "system"` really ought to work. Please just remove this comment.